config/settings/base.py

```diff
@@ -609,6 +609,18 @@
     },
 }
 
+# Time estimates for reviewing tasks based on the assigned story points
+#
+# SPRINT_HOURS_RESERVED_FOR_REVIEW {
+#     "0": Review time if task is not estimated,
```

Comment:
This is not true. Please make [this GET request](https://tasks.opencraft.com/rest/api/2/issue/BB-4594) and check the value of `customfield_10002` ("Story Points", as described [here](https://tasks.opencraft.com/rest/api/2/field)). It is `null`.
We could handle this by adding `null` as a dict key.

Reply:
Yes, now I read a null key from the dict